catroid/src/main/java/org/catrobat/catroid/ProjectManager.java

```diff
@@ -314,7 +314,7 @@ public void saveProject(Context context) {
 		}
 	}
 
-	public boolean initializeDefaultProject(Context context) {
+	public boolean initializeDefaultProject(final Context context) {
```

Comment:
> Things to avoid when creating commits
>
> * Mixing two unrelated functional changes.
> * …

If you fix a bug, always only add all necessary changes to the commit but nothing more.